docker-compose.yaml

```diff
@@ -165,6 +165,23 @@ services:
     depends_on:
       - geth
 
+  espresso-sequencer:
+    pid: host # allow debugging
+    image: espresso-integration-testnode
```

Comment:
I think this should be `ghcr.io/espressosystems/nitro-espresso-integration/nitro-node-dev:integration` eventually but it's currently necessary to use a local docker image for testing because we haven't merged @zacshowa's E2E changes in nitro and they will be needed for this upgrade test. So I guess it's a locally tagged image.